### ISSUE TITLE
TypeError: Cannot read property 'sid' of undefined

### ISSUE THREAD

**Issue by rhys-edwards** [twilio/twilio-node#126]
Being thrown the above error whilst trying to use the code from the Node.JS API explorer in Twilio dev tools. 

Error:
````
7 Oct 22:01:52 - [nodemon] starting `node scraper.js`
Free on Xbox One, Xbox 360, PS3, PS4: Tales from the Borderlands (Episode 1)
/Users/rhysedwards/Downloads/insightful/ozbargain/node_modules/twilio/node_modules/q/q.js:126
                    throw e;
                    ^

TypeError: Cannot read property 'sid' of undefined
    at /Users/rhysedwards/Downloads/insightful/ozbargain/scraper.js:22:31
    at /Users/rhysedwards/Downloads/insightful/ozbargain/node_modules/twilio/node_modules/q/q.js:1924:17
    at flush (/Users/rhysedwards/Downloads/insightful/ozbargain/node_modules/twilio/node_modules/q/q.js:108:17)
    at doNTCallback0 (node.js:408:9)
    at process._tickCallback (node.js:337:13)
7 Oct 22:01:54 - [nodemon] app crashed - waiting for file changes before starting...
````

Code:
````
var accountSid = 'A********';
var authToken = 'd*********';

var fs = require('fs'),
    request = require('request'),
    cheerio = require('cheerio'),
    client = require('twilio')(accountSid, authToken);

url = 'http://www.ozbargain.com.au';

request(url, function(error, response, html) {
    if (!error && response.statusCode == 200) {
        var $ = cheerio.load(html);
        var $el = $("a:contains('Xbox')");

        if ($el.length) {
          client.messages.create({
          	to: "+61448141065",
          	from: "+61418739508",
          	body: "hey"
          }, function(err, message) {
          	console.log(message.sid);
          });
            console.log($el.text());
        } else {
            console.log('hey');
        }
    }
});

````

**Comment by philnash** [comment]
Looks like you might be getting an error back, not the created message. Try changing your code to something like this and see if there is an error logged:

    if ($el.length) {
      client.messages.create({
        to: "+61448141065",
        from: "+61418739508",
        body: "hey"
      }, function(err, message) {
        if(err){
          console.log(err);
        } else {
          console.log(message.sid);
        }
      });
        console.log($el.text());
    } else {
        console.log('hey');
    }

**Comment by rhys-edwards** [comment]
Ah thank you, that did the trick. Will close this issue.

**Comment by chandankrishnan** [comment]
router.post('/send',function(req,res){
var client = require('twilio')(secret.accountSid, secret.authToken); 
    client.sendMessage({ 
                        to: to, 
                        from: from, 
                        body: "Hello",
                },function(err, message){ 
                    if(err)
                        console.log(err);
                        console.log(message.sid); 
                    });
});
in nodeJs with expressJs framework

**Comment by GiuseppeSaponieri** [comment]
Cannot read property 'message' of undefined

**Comment by childish-sambino** [comment]
@GiuseppeSaponieri Please open a new issue with details about the problem you're seeing.
